native/acter/src/api/news.rs

```diff
@@ -377,21 +377,24 @@ pub struct NewsEntryDraft {
 }
 
 impl NewsEntryDraft {
-    pub fn add_text_slide(&mut self, body: String) -> &mut Self {
+    pub fn add_text_slide(&mut self, body: String) {
+        trace!("add text slide");
+        let text_content = TextMessageEventContent::markdown(body);
+
         self.slides.push(NewsSlide {
             client: self.client.clone(),
             room: self.room.clone().into(),
-            inner: news::NewsSlide::new_text(body),
+            inner: news::NewsSlide {
+                content: news::NewsContent::Text(text_content),
+                references: Default::default(),
+            },
```

Comment:
Why do you move this NewsSlide creation part out of the `core` (which can be used without the dart) to the `api`? I think the internal format of `news::NewsSlide` for most of `api` should stay rather opaque and just use functions rather than know off and create it with all its fields...  Makes it a lot easier to change the internal structure of the core-types, if their primary way of being instantiated is through function calls..